advice/Gavin-Reid-advice.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/Gavin Reid.css">
```

Comment:
Hi @gjlreid,

You're super close! You added a link to your own CSS file in the <head> of your advice.html file. But the css file you linked to isn't in the css folder. If you've decided not to add one, you could change it back to the main.css file. Either way would be completely fine!

Best,
Lisa